### PR TITLE
Schematic design diagram

### PR DESCRIPTION
**Pull request by msvinaykumar**
This PR created to draw Schematic design diagram representing each block diagram and interconnection using draw.io

### CODE REVIEW

File: design/ExperimentManager.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  <img src="/design/images/EmOnly.png">
+</p>
+
+## EM Architecture
+The experiment manager contains the following main building blocks:
+
+- ### Building blocks.
+  - Dispatcher handler
+    - This block is used to accept input requests in the form of JSON format, which contains experimental tunables along with evaluation metrics queries. This handler is used with the Taskmanager handler to run multiple trials in parallel per deployment.
+  - TaskManger
```
Comment (rbadagandi) [inline design/ExperimentManager.md:60]: Task Manager

File: design/ExperimentManager.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    - This block is designed as an event-driven architecture where RestAPI will produce trials and store them in a queue. And there will be a scheduler running for every specific given delay to check if any messages are in queue and trigger the Iteration Manager if any new trials are found.
+  - IterationManger
+    - This block is used to perform primary tasks such as
+      Deploy and gather metrics for a specified number of iterations.
+      Following the completion, summarise the metrics results and send them to the recommendation manager or user. The Recommendation Manager further evaluates the metrics results and suggests more trials if needed.
+  - Target Handler
+    - This block contributes to the provision of a wrapper API around the Kubernetes Java client for running experiment trials on the Kubernetes environment.
+  - Datasource Handler
+    - These blocks help in the collection of metrics results using data sources like Promethus, Instana, Datadog etc.
```
Comment (rbadagandi) [inline design/ExperimentManager.md:69]: Prometheus

**Comment by msvinaykumar** [comment]
Review comments pending

**Comment by dinogun** [comment]
Closing as this is stale. Re-open if needed.

**Pull request closed without merging**